src/aleph/vm/pool.py

```diff
@@ -161,6 +161,12 @@ def get_unique_vm_id(self) -> int:
             # anymore.
             currently_used_vm_ids = {execution.vm_id for execution in self.executions.values() if execution.is_running}
             for i in range(settings.START_ID_INDEX, 255**2):
+
+                if self.network:
+                    # Check the network interface don't already exists, otherwise it will cause a crash
+                    if self.network.interface_exists(i):
+                        continue
```

Comment:
Would it make more sense to delete the interface and then recreating it with the proper configuration ?